### ISSUE TITLE
Better embedding

### ISSUE THREAD

**Issue by xplosionmind** [manunamz/jekyll-wikilinks#30]
As of right now, when I intend to embed a note by using `![[example note]]` what I get is the title of the linked page and below all of its content.

There is no `<a>` tag for the title of the linked page! There should be…

**Comment by manunamz** [comment]
I'm not sure if I understand -- are you saying you're expecting the title of the embedded file to be a link to that file's page?

**Comment by xplosionmind** [comment]
> are you saying you're expecting the title of the embedded file to be a link to that file's page?

Exactly, in such a way, that it is clear that there is a reference to a note, and otherwise it seems just a differently styled content. Furthermore, it would be nice to give the option to generate a heading with the name of the referenced file / subsection, so that it is more visible as a section of the parent note.
